app/src/main/java/com/example/trainingintive/dog_images_feature/presentation/AlbumItemTouchHelper.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.trainingintive.dog_images_feature.presentation
+
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+
+class AlbumItemTouchHelper(
+    private val onMove: (Int, Int) -> Unit,
+    private val onSwiped: (Int) -> Unit
+) {
+
+    private val helper = ItemTouchHelper(object : ItemTouchHelper.SimpleCallback(
+        ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT or
+            ItemTouchHelper.DOWN or ItemTouchHelper.UP,
+        ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT
```

Comment:
Extract these directions to some private object, const, or somewhere else. You should give them some meaningful names.